ui/components/app/snaps/insight-warnings/insight-warnings.js

```diff
@@ -149,12 +155,14 @@ export default function InsightWarnings({
           }}
         >
           <Checkbox
-            variant={TextVariant.bodySm}
+            variant={TextVariant.bodyMd}
             isChecked={isChecked}
             onChange={handleOnChange}
             label={t('insightWarningCheckboxMessage', [
               t(InsightWarningLanguage[action].imperative),
-              stripHttpSchemes(origin),
+              <span key={action}>
+                <b>{stripHttpSchemes(origin)}</b>
+              </span>,
```

Comment:
```suggestion
              <Text as="span" key={action] fontWeight={FontWeight.Medium}>
                {stripHttpSchemes(origin)}
              </Text>
```

Same suggestion applies elsewhere